edb/schema/annos.py

```diff
@@ -70,6 +70,15 @@ class AnnotationValue(
     value = so.SchemaField(
         str, compcoef=0.909)
 
+    # Overloaded to mark it as not inheritable, since I found it
```

Comment:
Hm, why is it `inheritable` in the base class in the first place?

Reply:
Having looked a bit I can't tell why it exists in the first place (apart from on annotation values).

There is syntax for declaring scalar types as FINAL, but as far as I could tell that doesn't really do anything? You can still subtype a final scalar. And there is syntax for doing SET FINAL as part of an alter on a wide variety of different objects, but mostly nothing good seems to happen if you do that. You can put it on a pointer, and then it will be left out when a subtype is created, and then there will be a crash when trying to execute the DDL? I'm guessing this is a vestigal feature?

The logic for how non-inheritable annotations are left out of children is based on it and lives in `inheriting`, which is I guess one reason why it it needs to exist on `SubclassableObject`. (At least absent some refactoring)